src/views/IdentityOverview/IdentityOverview.tsx

```diff
@@ -52,23 +49,14 @@ export function IdentityOverview({ identity }: Props): JSX.Element | null {
   const showDownloadPrompt =
     credentials && credentials.some(({ isDownloaded }) => !isDownloaded);
 
-  const hasLegacyDid = useSwrDataOrThrow(
-    did,
-    needLegacyDidCrypto,
-    'needLegacyDidCrypto',
-  );
+  const hasLegacyDid = useAsyncValue(needLegacyDidCrypto, [did]);
```

Comment:
Would a hook like `useNeedLegacyDidCrypto` make it more clear in the test what exactly is being mocked?